docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -1,4 +1,117 @@
 Relative Hybrid Topology Protocol
 =================================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The relative free energy calculation approach calculates the difference in 
+free energy between two similar ligands. Depending on the ``ChemicalSystem`` 
```

Comment:
```suggestion
free energy between two similar ligands. Depending on the :class:`.ChemicalSystem` 
```
We probably should come up with standard for this, I generally like to point to the class whereever I can, but maybe it's just me.